styles.css

```diff
@@ -1,138 +1,601 @@
 body {
   text-transform: capitalize;
-  overflow: hidden;
   background-color: #ffff;
   box-sizing: border-box;
-  overflow: scroll
+  overflow: scroll;
 }
 
 @import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
 
 * {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
   font-family: "Poppins", sans-serif;
 }
 
-@media (min-width: 375px) {
-  .nav-bar {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    font-size: 19px;
-  }
+.container {
+  margin: auto 2.4rem;
+  overflow: hidden;
 }
 
-@media (min-width: 768px) {
-  .nav-bar {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    font-size: 19px;
-  }
+header {
+  background-color: #fff;
+}
 
-  body {
-    background-color: #fff;
-  }
+html {
+  font-size: 62.3%;
+  scroll-behavior: smooth;
 }
 
 .nav-bar {
   display: flex;
   justify-content: space-between;
-  top: 37px;
-  bottom: 30px;
-  z-index: 1;
+  align-items: center;
+  width: 100%;
+  height: 10vh;
+  padding: 3%;
+  top: 0;
 }
 
-nav a {
-  padding-right: 3%;
+.list-nav {
+  display: none;
 }
 
-.logo a {
-  font-size: 1.2rem;
-  font-weight: 700;
+.link {
+  margin-bottom: 3.4rem;
 }
 
-.link nav a {
+a {
   text-decoration: none;
-  display: inline-block;
-  flex-direction: row;
+}
+
+.logo a {
+  font-size: 1.2rem;
+  font-weight: 700;
 }
 
 .logo {
-  margin-top: 20px;
   color: #6070ff;
-  font-size: 16px;
+  font-size: 1.8rem;
+  font-weight: 700;
+  text-decoration: none;
 }
 
-.nav-bar .logo {
-  flex-wrap: wrap;
-  flex: 1;
-  order: none;
-  margin-left: 10px;
+.container-all {
+  display: flex;
+  flex-direction: column;
+  background-image: url('./images/header.now.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 100% 9rem;
+  padding-bottom: 1rem;
+  margin-bottom: 12%;
+  background-color: #ffff;
+  border-bottom-left-radius: 4pc;
 }
 
-.dashboard {
-  border: 0 solid #acb7c3;
-  color: #6070ff;
-}
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 12.7rem;
+  background-color: #f7f7f7;
+  padding: 12.8rem 0 2.8rem 0;
 
-.nav-bar .dashboard {
-  margin-right: 10px;
-  margin-top: 20px;
+  /* padding-bottom: 12.8rem; */
+  margin-bottom: 12.8rem;
 }
 
-.container {
+.container1 {
   display: flex;
-  flex-wrap: wrap;
   flex-direction: column;
-  background-image: url('./images/header-shapes\ mobile.png');
+  margin-top: 10%;
+  margin-bottom: 0%;
+  background-size: cover;
   background-repeat: no-repeat;
-  background-size: cover; 
-  justify-content: center;
-  
+  background-position: 100%;
+  background-image: url('./images/header-shapes\ mobile.png');
 }
```

Comment:
- [ ] Please make your bottom left corner rounded to match the figma design template.

|Figma design|Your design|
|---|-----|
|<img width="322" alt="Screenshot 2023-05-29 at 10 33 17 PM" src="https://github.com/hetrox8/portfolio1/assets/44798044/bffbb043-db08-490f-9b16-02da974af957">|<img width="366" alt="Screenshot 2023-05-29 at 10 33 43 PM" src="https://github.com/hetrox8/portfolio1/assets/44798044/6f25d7c7-0d7c-4b96-adc7-0daffa53da01">|